Midi alt design

s-zaj opened the pull request:
- use modern CMOS optocoupler that is faster, smaller, cheaper and does not need us to fiddle with a pull-up resistor
- add RX/TX activity monitor LEDs for easier debugging
- flip midi_in polarity labels

Commit:
-Add MIDI electrical and TRS spec
-Add newer optocoupler datasheet

Jaxc commented on PCB/Magna/MCU.kicad_sch:
Is there an actual change in this file? Otherwise lets remove it to keep the log a bit more clear :)

Jaxc commented on PCB/Magna/MIDI.kicad_sch:
Is there an actual change in this file? Otherwise lets remove it to keep the log a bit more clear :)

Jaxc commented on PCB/Magna/Magna.kicad_pcb:
Is there an actual change in this file? Otherwise lets remove it to keep the log a bit more clear :)

Jaxc commented on PCB/Magna/Magna.kicad_sch:
Is there an actual change in this file? Otherwise lets remove it to keep the log a bit more clear :)

Jaxc commented on PCB/Magna/Power.kicad_sch:
Is there an actual change in this file? Otherwise lets remove it to keep the log a bit more clear :)

Jaxc commented on PCB/Magna/Usb.kicad_sch:
Is there an actual change in this file? Otherwise lets remove it to keep the log a bit more clear :)

Jaxc commented on PCB/Magna/human_interface.kicad_sch:
Is there an actual change in this file? Otherwise lets remove it to keep the log a bit more clear :)

Jaxc commented on PCB/Magna/power_led.kicad_sch:
Is there an actual change in this file? Otherwise lets remove it to keep the log a bit more clear :)

s-zaj commented:
Wasn’t actually sure what to do about the autochanged files (thought maybe the kicad project would break if they were not committed together?) but let’s try

Jaxc commented:
It seems to be working fine with just the one file changed!

Commit:
-Add MIDI alternative design
-substitute CMOS optocoupler TLP2361. This is faster, smaller, cheaper and does not need pull-up resistor to VDD.
-add RX/TX activity monitor LEDs for debugging
-flip midi_in+ and midi_in-

Commit:
-Add MIDI pwl stream generator script
-Add LTSpice sim for RX/TX LED indicators (uses the generated pwl file as input)
-Update R value for filter in schematic after simulations

Commit:
-Add LCSC numbers
-Fix annotation and presentation
-update .gitignore

Commit:
substitute MIDI sheets and click save
